tests/hook_implementations/test_hook_implementations.py

```diff
@@ -3,11 +3,23 @@
 from ert.shared.plugins.plugin_manager import ErtPluginManager
 
 import semeio.hook_implementations.forward_models
+from semeio.forward_models import Design2Params, DesignKW, Pyscal, GenDataRFT, OTS
 from semeio.workflows.ahm_analysis import ahmanalysis
 from semeio.workflows.csv_export2 import csv_export2
 from semeio.workflows.localisation import local_config_script
 
 
+def test_that_installable_fm_steps_work_as_plugins():
+    plugin_manager = ErtPluginManager()
+    fms = plugin_manager.forward_model_steps
+
+    assert Design2Params in fms
+    assert DesignKW in fms
+    assert Pyscal in fms
+    assert GenDataRFT in fms
+    assert OTS in fms
```

Comment:
I assume it was done by design to omit `InsertNoSim`, `RemoveNoSim` & `ReplaceStringConfig` ?

Reply:
Actually nice catch, I think these omited values are actually meant to be included